developmentalparsers/w2dynamics/parser.py

```diff
@@ -139,5 +139,5 @@ def parse_quantities(source, target, include=[]):
             for atom in self.parameter_parser.get('atom', []):
                 parameters = {key: val for key, val in atom.get('parameter', [])}
                 if parameters:
-                    sec_atom = sec_method.m_create(x_w2dynamics_atom)
-                    sec_atom.x_w2dynamics_atom_parameters = parameters
+                    sec_atom = sec_method.m_create(x_w2dynamics_atom_parameters)
```

Comment:
Maybe, we could also simply extend the section AtomParameters in the general nomad metainfo Method section

Reply:
I will soon push another set of changes, working on a new `class DMFT(MSection)` in which I include these things. This will be incomplete, thought, as we also need to parse the w2d_Hk.dat files that are provided with uploads for crystals.

I have an extra plan, which is to work on System and create some class to include models (apart from crystals). I will get back to Luca and you about this.